package.json

```
@@ -42,7 +43,7 @@
     "start": "node src/index.js",
     "lint": "prettier --check \"src/**/*.js\" \"src/**/*.css\"",
     "prettier:write": "prettier --write \"src/**/*.js\" \"src/**/*.css\"",
-    "postinstall": "cp node_modules/prismjs/themes/*.css src/static/css/prismjs",
+    "postinstall": "cpy 'node_modules/prismjs/themes/*.css' 'src/static/css/prismjs/'",
```

```suggestion
    "postinstall": "node -e 'fs.cpSync(\"./node_modules/prismjs/themes/\", \"./src/static/css/prismjs/\", { recursive: true })'",
```

We might be able to get away with something like this without installing any additional deps?